src/main/antlr/GraphqlSDL.g4

```diff
@@ -59,7 +59,7 @@ objectTypeExtensionDefinition :
 ;
 
 implementsInterfaces :
-    IMPLEMENTS '&'? typeName+ |
+    IMPLEMENTS '&'? typeName |
     implementsInterfaces '&' typeName ;
```

Comment:
my antlr is not always top notch but how does this repeat

```
interface x implements a & b & c 
```
Seems like it would only allow one lot of "implements name" and then one lot of `& tname`

Reply:
The definition is recursive. For this example, `implements a & b & c` would be broken down as

1. `implements a` matches the rule `IMPLEMENTS '&'? typeName`, so this is a `implementsInterfaces` block
2. Then `implements a & b` could be compressed into `implementsInterfaces & b`, which matches the second rule `implementsInterfaces '&' typeName`. This could be considered another `implementsInterfaces` block
3. Then `implements a & b & c` could be compressed into `implementsInterfaces & c`, which matches the second rule again